packages/terra-consumer-icon/scripts/src/generate-icon/Icon.js

```diff
@@ -0,0 +1,17 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
https://github.com/cerner/terra-core/blob/master/packages/terra-icon/scripts/src/generate-icon/Icon.js
also the same